### ISSUE TITLE
ArtifactElectricityTrigger can never actually trigger

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#14506]
It subscribes to 1 event that is never raised.

**Comment by metalgearsloth** [comment]
Nope. There's code in stunbatonsystem but it's never actually run.

**Comment by Myakot** [comment]
Multitool-LMB is an activator for an artifact, same with activated stunbaton. If there's still doubt I'll record and post it.

**Comment by ThatGuyUSA** [comment]
We have new Xeno archeology #33370

Formerly known as an electricity trigger, has now been made "pulse" which is done through a multi tool.
It's called `TriggerPulsing` and it's in the file `Resources/Prototypes/XenoArch/triggers.yml`
